src/pages/home/sidebar/OptionRow.js

```diff
@@ -81,204 +81,213 @@ const defaultProps = {
     disableRowInteractivity: false,
 };
 
-const OptionRow = (props) => {
-    const textStyle = props.optionIsFocused
```

Comment:
I think we can revert this file back to the functional component and the ref will still work. Could you please revert it and test it?

Reply:
> I think we can revert this file back to the functional component and the ref will still work. Could you please revert it and test it?

I works inside a functional component, yes. 
But in[ your comment ](https://github.com/Expensify/App/issues/6470#issuecomment-1022138017)you have mentioned that we don't use hooks:
> First, we don't use hooks https://github.com/Expensify/App/blob/main/STYLE.md#stateless-components-vs-pure-components-vs-class-based-components-vs-render-props---when-to-use-what. You will have to convert that to the class component.